### ISSUE TITLE
should the instructor teaching philosophy be in lesson?

### ISSUE THREAD

**Issue by wikfeldt** [coderefinery/instructor_training#22]
referring to this: https://coderefinery.github.io/instructor_training/01-intro/#anne-fouilloux

if all instructors add something there it might get very long... Maybe we can either remove it from the lesson and add elsewhere (hackpad) or just talk about it, or keep it in lesson but make it expandable (in "challenge" format). Thoughts @annefou ?

**Comment by annefou** [comment]
Yes. In fact I would like to have it on the coderefinery website and a link in the instructor training material. The "About" section presents each member of the team but we do present ourselves as individual not as member of the CodeRefinery community. I think it would be nice to have a section on CodeRefinery training philosophies or similar where we could add these "stories". Then it would be easier to talk about (and build) CodeRefinery community.

**Comment by wikfeldt** [comment]
that's a great idea! the "about" page on coderefinery.org would probably need some structural changes to make that look good

**Comment by annefou** [comment]
Yes. That is why I just added it in the instructor training material... I was not sure how to present it yet.

**Comment by bast** [comment]
Do I understand that what is left in this issue is to link this from https://coderefinery.org/about/? If yes, let's move this issue to the other repo.

**Comment by wikfeldt** [comment]
yes, move this issue to coderefinery.org repo
